mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +103,238 @@ objects:
+              Only supported for custom-trained Models and AutoML Tabular Models.
+          - !ruby/object:Api::Type::String
+            name: 'enableAccessLogging'
+            description: |
+              These logs are like standard server access logs, containing information like timestamp and latency for each prediction request.  
+              Note that Stackdriver logs may incur a cost, especially if your project receives prediction requests at a high queries per second rate (QPS). Estimate your costs before enabling this option.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'privateEndpoints'
+            input: false
+            output: true
+            description: |
+              Provide paths for users to send predict/explain/health requests directly to the deployed model services running on Cloud via private services access. This field is populated if network is configured.  
+            properties:  
+              - !ruby/object:Api::Type::String
+                name: 'predictHttpUri'
+                description: Http(s) path to send prediction requests.
+                input: false
+                output: true
+              - !ruby/object:Api::Type::String
+                name: 'explainHttpUri'
+                description: Http(s) path to send explain requests.
+                output: true
+                input: false
+              - !ruby/object:Api::Type::String
+                name: 'healthHttpUri'
+                description: Http(s) path to send health requests.
+                input: false
+                output: true
+          - !ruby/object:Api::Type::NestedObject
+            name: 'dedicatedResources'
+            description: |
+              A description of resources that are dedicated to the DeployedModel, and that need a higher degree of manual configuration.  
+              Cannot be used with `automaticResources`
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'MachineSpec'
+                required: true
+                description: |
+                  Immutable. The specification of a single machine used by the prediction.
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'machineType'
+                    description: |
+                      Immutable. The type of the machine.  
+                      See the [list of machine types supported for prediction](https://cloud.google.com/vertex-ai/docs/predictions/configure-compute#machine-types)  
+                      See the [list of machine types supported for custom training](https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types).  
+                      For `DeployedModel` this field is optional, and the default value is `n1-standard-2`. For `BatchPredictionJob` or as part of `WorkerPoolSpec` this field is required.  
+                  - !ruby/object:Api::Type::Enum
```

Comment:
Rather than use an enum, let's use a string and list the currently supported types in the description. The list of accelerator types tends to change over time, strongly typed enums aren't appropriate for that as a new value will need a code change + provider update.